### ISSUE TITLE
weried phenomenon about using main.py train ucf101 dataset

### ISSUE THREAD

**Issue by dandingol03** [yjxiong/tsn-pytorch#122]
Hi, @yjxiong 
 i use the main.py  script to train ucf101 , but it seems like the script is just loading the rawframe, and never begin to train the model. Here is my setting:

python main.py ucf101 RGB <ucf101_rgb_train_list> <ucf101_rgb_val_list> \
   --arch BNInception --num_segments 3 \
   --gd 20 --lr 0.001 --lr_steps 30 60 --epochs 80 \
   -b 120 -j 8 --dropout 0.8 \
   --snapshot_pref ucf101_bninception_ 

Thanks in advance~

**Comment by dandingol03** [comment]
i figure out why, because i use the google colab, it needs a long time to load data. After that, it begin to train
